src/pages/login/Login.jsx

```diff
@@ -1,40 +1,36 @@
 import React, { useState } from "react";
 import { useNavigate } from "react-router-dom";
-import { useForm } from "react-hook-form";
 
 import axios from "axios";
-
-import { isEmailValid, isPasswordValid } from "../../utils/validate";
+import { useForm } from "react-hook-form";
 
 import { useAuth } from "../../context/AuthContext";
 
-import { API_END_POINT } from "../../../config";
 
-import dckapPalliLogo from "/images/dckap_palli_logo_lg.svg";
-import managerLoginLogo from "/images/manager_login_image.svg";
+import { isEmailValid, isPasswordValid } from "../../utils/validate";
```

Comment:
Is there no uttitlites provided by react-hook-form to handle these kind of errors, instead of us manually handling it.

Reply:
It would be better, if react-hook form handle errors